aws/templates/solution/solution_ecs.ftl

```diff
@@ -53,6 +53,47 @@
             name=formatComponentLogGroupName(tier, component) /]
     [/#if]
 
+    [#if deploymentSubsetRequired("efs", true) && 
+            ecs.ClusterWideStorage?has_content && ecs.ClusterWideStorage ]
+
+        [#assign ecsEFSVolumeId = formatEFSId(tier, component)]
+        [#assign ecsEFSVolumeName = formatName( tier, component, "efs")]
```

Comment:
Doesn't need efs in the name I think - it will be shown in the console under EFS - but it should be the fulkl component name including product/segment

Reply:
Name updated to formatFullComponentName and "efs" removed